ios/FluentUI/Vnext/Avatar/Avatar.swift

```diff
@@ -86,6 +86,25 @@ public struct AvatarView: View {
         self.tokens = state.tokens
     }
 
+    /// Creates and initializes a SwiftUI Avatar
+    /// - Parameters:
+    ///   - style: The style of the avatar.
+    ///   - size: The size of the avatar.
+    ///   - image: The optional image that the avatar should display.
+    ///   - primaryText: The primary text used to calculate the Avatar initials.
+    ///   - secondaryText: The secondary text used to calculate the Avatar initials.
+    public init(style: MSFAvatarStyle,
+                size: MSFAvatarSize,
+                image: UIImage?,
```

Comment:
We can combine this with the other init by giving these three optionals default `nil` values.